controllers/flinkcluster_converter.go

```diff
@@ -562,7 +562,7 @@ func getDesiredConfigMap(
 			Namespace: clusterNamespace,
 			Name:      configMapName,
 			OwnerReferences: []metav1.OwnerReference{
-				ToOwnerReference(flinkCluster)},
+				ToJobOwnerReference(flinkCluster)},
```

Comment:
I don't quite understand why we call it `ToJobOwnerReference` in a `ConfigMap` resource? What is the consequence of setting `BlockOwnerDeletion=true` for it? Will I still be able to delete the FlinkCluster CR?

Reply:
commit error